res/css/style.css

```diff
@@ -220,45 +219,52 @@ body, #root, .profileViewer {
   color: transparent;
 }
 
-.treeRowToggleButton {
-  background-image: url(./treetwisty.svg);
-}
-
 .focusCallstackButton {
-  background-image: url(./circlearrow.svg);
+  background-image: url(../img/svg/circlearrow.svg);
```

Comment:
I'm not a big fan of these subdirectories `svg`/`png`/`jpg` (I'm fine with `/img`).. I'd rather have subdirectories for concerns: like `calltree` here, `search` for the looking glass, etc. But maybe we don't even need more subdirectories after `img`. What do you think ?

Reply:
I'd say, I get your point and I agree, we should split both up. But as this will be changed anyway, lets keep it as I am pretty packed with work right now and file a follow up issue to optimize this (move to components including css). Otherwise this PR might be sitting for longer. :/

What do you think?

Reply:
Good for me. If this bothers me too much I'll do it myself, but I don't expect it ;) After all we don't touch the images so much...